clients/src/main/resources/common/message/ElectLeadersResponse.json

```diff
@@ -16,11 +16,13 @@
 {
   "apiKey": 43,
   "type": "response",
-  "name": "ElectPreferredLeadersResponse",
-  "validVersions": "0",
+  "name": "ElectLeadersResponse",
+  "validVersions": "0-1",
   "fields": [
     { "name": "ThrottleTimeMs", "type": "int32", "versions": "0+",
       "about": "The duration in milliseconds for which the request was throttled due to a quota violation, or zero if the request did not violate any quota." },
+    { "name": "ErrorCode", "type": "int16", "versions": "1+", "ignorable": false,
```

Comment:
Do you think we may as well add an error message field as well?

Reply:
I was wondering the same thing. I got inspiration from `FetchRequest`. There is no error message there so I figure I would not add it for consistency.

I'll add it if we think that every request should have `ThrottleTimeMs`, `ErrorCode` and `ErrorMessage` at the top level.

Reply:
I don't have a strong opinion. It would be nice to refactor the protocols to have a standard set of top level fields. Since we don't have a strong reason for the field now, maybe we can leave it for the future.